src/main/java/org/jabref/logic/importer/fetcher/ScholarArchiveFetcher.java

```diff
@@ -0,0 +1,167 @@
+
+            // Parse the JSON response into a list of BibEntry objects.
+            JSONObject jsonObject = new JSONObject(response);
+            List<BibEntry> entries = new ArrayList<>();
+            if (jsonObject.has("results")) {
+                JSONArray results = jsonObject.getJSONArray("results");
+                for (int i = 0; i < results.length(); i++) {
+                    JSONObject jsonEntry = results.getJSONObject(i);
+                    BibEntry entry = parseJSONtoBibtex(jsonEntry);
+                    entries.add(entry);
+                }
+            }
+
+            return entries;
+        };
+    }
+
+
+    @Override
+    public String getName() {
+        return FETCHER_NAME;
+    }
+
+    private BibEntry parseJSONtoBibtex(JSONObject jsonEntry) throws ParseException {
+        try{
+        BibEntry entry = new BibEntry();
+        EntryType entryType = StandardEntryType.InCollection;
+        JSONObject biblio = jsonEntry.optJSONObject("biblio");
+        JSONObject abstracts = jsonEntry.optJSONObject("abstracts");
+
+            // publication type
+            String type = biblio.optString("release_type");
+            entry.setField(StandardField.TYPE, type);
+            if (type.toLowerCase().contains("book")) {
+                entryType = StandardEntryType.Book;
+            } else if (type.toLowerCase().contains("article")) {
+                entryType = StandardEntryType.Article;
+            }
+            entry.setType(entryType);
+
+
+            entry.setField(StandardField.TITLE, biblio.optString("title"));
+            entry.setField(StandardField.JOURNAL,biblio.optString("container_name"));
+            entry.setField(StandardField.DOI,biblio.optString("doi"));
+            entry.setField(StandardField.ISSUE,biblio.optString("issue"));
+            entry.setField(StandardField.LANGUAGE,biblio.optString("lang_code"));
+            entry.setField(StandardField.PUBLISHER,biblio.optString("publisher"));
+
+            entry.setField(StandardField.YEAR, String.valueOf(biblio.optInt("release_year")));
+            entry.setField(StandardField.VOLUME,String.valueOf(biblio.optInt("volume_int")));
+            entry.setField(StandardField.ABSTRACT,abstracts.optString("body"));
+
+            // Date
+            String dateString = (String) biblio.get("date");
+
+            entry.setField(StandardField.DATE,dateString);
+
+            // Authors
+            if (biblio.has("contrib_names")) {
+                JSONArray authors = jsonEntry.getJSONArray("contrib_names");
+                List<String> authorList = new ArrayList<>();
+                if(authors!=null && authors.length()>0){
+                    for (int i = 0; i < authors.length(); i++) {
+                        authorList.add(authors.getString(i));
+                    }
+                }
+                entry.setField(StandardField.AUTHOR, String.join(" and ", authorList));
```

Comment:
Concatenation with and is okay, but for the field use 
Authorlist.parse(....).getAsFirstLastNamesWithAnd();